face_track/face_track.py

```diff
@@ -376,15 +378,16 @@ def face_event_cb(self, data):
 	def face_loc_cb(self, data):
 		if not self.control_mode & self.C_FACE_TRACKING:
 			return
-		for face in data.faces:
-			fid = face.id
-			loc = face.point
-			# Sanity check.  Sometimes pi_vision sends us faces with
-			# location (0,0,0). Discard these.
-			if loc.x < 0.05:
-				continue
-			self.add_face(fid)
-
+		#below causes error in cmt .. mandeep
```

Comment:
OK, well, if you read the comments in the code surrounding this, you will see a suggestion that `face_event_cb` is probably not needed, and should be removed someday ... and that the face_loc_cb was added precisely to work around the bug with the spurious face-lost messages. So the change that you made is exactly the opposite of what the comments in the code are saying.

Its OK to make the change that you did -- but its incomplete!   If CMT does not  have this bug, that pi_vision did, then the right answer is to remove the face_loc_cb callback entirely, instead of stubbing it out, without any explanation.  At the same time, the comments on `face_event_cb` should be updated to reflect the new reality -- that they are the primary source of face enter/leave events.

Another issue is that the code refers to pi_vision everywhere.  If we are certain about the conversion to CMT (are we??) then all the references to pi_vision should be reworded so that they say "the right thing" instead of saying things that are wrong/misleading. 

Finally, if we want to keep the code backwards compatible so that it works with both pi_vision, and with CMT for w while .. then it should be made backwards compatible.   If it does not need to be backwards compatible, then don't leave behind stubs and broken comments and other garbage in the code.